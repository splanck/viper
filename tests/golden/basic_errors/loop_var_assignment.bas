10 FOR I = 1 TO 3
20 LET I = 2
30 NEXT I
