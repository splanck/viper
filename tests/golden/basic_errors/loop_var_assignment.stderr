B1010.*cannot assign to loop variable 'I' inside FOR
